JamesRobertSutcliffe/personality-music-recommender
# Set up local PostgresSQL database

This PR establishes the groundwork for upcoming changes concerning the database and development environment. 

The changes ensure all necessary dependencies are incorporated into the project to enable the usage of a local database for testing purposes. 
Additionally, it makes use of shell scripting to automate development tasks associated with initialising and configuring the project.

### Testing
1. Ensure you have [Docker](https://www.docker.com/) installed
2. Run `npm i` to install the dependencies
3. Run the `./scripts/setup.sh` command
4. Run the `./scripts/server.sh` command
5. On a separate terminal instance, enter the `npm run test` command
6. Confirm you were able to do all of the above without issues

<details>
<summary>Related issues</summary>
Closes #15 
Closes #16 
Closes #21 
Closes #23 
Closes #24 
Closes #25 
</details>
